src/main/java/org/opensearch/flowframework/model/Config.java

```diff
@@ -21,29 +23,46 @@
 import static org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken;
 import static org.opensearch.flowframework.common.CommonValue.CREATE_TIME;
 import static org.opensearch.flowframework.common.CommonValue.MASTER_KEY;
+import static org.opensearch.flowframework.common.CommonValue.TENANT_ID_FIELD;
 
 /**
  * Flow Framework Configuration
  */
+
 public class Config implements ToXContentObject {
 
+    private static final Logger logger = LogManager.getLogger(Config.class);
     private final String masterKey;
     private final Instant createTime;
+    private final String tenantId;
 
     /**
      * Instantiate this object
      *
-     * @param masterKey The encryption master key
+     * @param masterKey  The encryption master key
      * @param createTime The config creation time
+     * @param tenantId   The tenantId
      */
-    public Config(String masterKey, Instant createTime) {
+    public Config(String masterKey, Instant createTime, String tenantId) {
         this.masterKey = masterKey;
         this.createTime = createTime;
+        this.tenantId = tenantId;
```

Comment:
Given all the complexities of potentially null tenant ID and the fact that we don't really ever use the value, can we just simplify here and only handle the case where the field exists, then read the value (`.objectTex()`) and ignore it?

Reply:
Also https://github.com/opensearch-project/flow-framework/blob/main/src/test/java/org/opensearch/flowframework/model/ConfigTests.java needs to be updated to test for present, null, and absent tenant Id